app/controllers/evaluations_controller.rb

```diff
@@ -143,7 +143,9 @@ def fetch_evaluator_submission_assignment
   end
 
   def can_access_evaluation?
-    @evaluator_submission_assignment && @evaluator_submission_assignment.user_id == current_user.id
+    evaluator_access = @evaluator_submission_assignment && @evaluator_submission_assignment.user_id == current_user.id
+    challenge_manager_access = @evaluator_submission_assignment.submission.challenge.challenge_manager_users.include?(current_user)
```

Comment:
Chris is working on the challenge manager ability to override scores but (I think) it won't work like this and so this will cause conflicts